src/drive/web/modules/trash/Toolbar.jsx

```diff
@@ -50,7 +50,11 @@ const Toolbar = ({
           {isMobile && (
             <>
               <ActionMenuItem
-                onClick={() => emptyTrash()}
+                onClick={() =>
+                  flag('drive.client-migration.enabled')
+                    ? client.collection('io.cozy.files').emptyTrash()
```

Comment:
Naive question : Is the query representing the trash correctly updated here ? How ?

Reply:
See your comment https://github.com/cozy/cozy-drive/pull/2044#discussion_r446298402 :D 

We receive the `DELETE_DOCUMENT` from the realtime. And then, we add the `_deleted:true` attribute to the file, and CozyClient has a build in mechanism to hide those documents with `deleted: true`

Reply:
I find it brittle to rely on the realtime for this, I find it a bit of a hack. Suppose we turn it off after a little bit of idle time, since it can be costly to keep a realtime connection to our servers, and then the trash would not work. In one of my PRs I had to do an adhoc receiveQueryResult to correctly keep the client store up-to-date. It's not perfect but I think I'd prefer it over the reliance on the realtime.

Here the main thing that bothers me is is that it's not very explicit that we rely on the realtime to keep cozy-client's collection updated.